browser/ui/webui/basic_ui.h

```diff
@@ -21,16 +21,25 @@ class WebUI;
 
 class Profile;
 
+#ifndef GRIT_RESOURCE_MAP_STRUCT_
+#define GRIT_RESOURCE_MAP_STRUCT_
+struct GritResourceMap {
+  const char* name;
+  int value;
+};
+#endif // GRIT_RESOURCE_MAP_STRUCT_
```

Comment:
Couldn't see a nicer way to include this header definition other than copy it here. Unlike other similar patterns in Chromium, this abstract class needs to know the specifics of this struct, since it uses the members, but does not itself include a specific implementation.